packages/fxa-auth-server/lib/routes/devices-and-sessions.js

```diff
@@ -351,8 +355,10 @@ module.exports = (
 
         return {
           enqueued: true,
-          notified: !!notifyError,
-          notifyError: notifyError,
+          notified: !notifyError,
```

Comment:
This was just straight-up wrong with the `!!`; thank you, additional test coverage!

Reply:
Hah I remember seeing this but for some reason thought you wanted `!!notifyError`🤦 Sorry about not commenting on that!